Symfony/CS/Fixer/Contrib/OrderedUseFixer.php

```diff
@@ -13,19 +13,66 @@
 namespace Symfony\CS\Fixer\Contrib;
 
 use Symfony\CS\AbstractFixer;
+use Symfony\CS\ConfigurationException\InvalidFixerConfigurationException;
 use Symfony\CS\Tokenizer\Tokens;
 
 /**
  * @author Sebastiaan Stok <s.stok@rollerscapes.net>
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ * @author Darius Matulionis <darius@matulionis.lt>
  * @author SpacePossum
  */
 class OrderedUseFixer extends AbstractFixer
 {
     const IMPORT_TYPE_CLASS = 1;
+
     const IMPORT_TYPE_CONST = 2;
+
     const IMPORT_TYPE_FUNCTION = 3;
 
+    const SORT_ALPHA = 'alpha';
+
+    const SORT_LENGTH = 'length';
+
+    /**
+     * Sorting type.
+     *
+     * @var string
+     */
+    private static $sortType = self::SORT_ALPHA;
+
+    /**
+     * Array of supported sort types.
+     *
+     * @var string[]
+     */
+    private static $supportedSortTypes = array(self::SORT_ALPHA, self::SORT_LENGTH);
+
+    /**
+     * @param array $sortType
+     */
+    public static function configure(array $sortType = null)
```

Comment:
that's the thing - the user will be able to use his order, indeed, any combination if `IMPORT_TYPE_*`